diff -up crda-3.13/Makefile.ldconfig-remove crda-3.13/Makefile
--- crda-3.13/Makefile.ldconfig-remove	2016-02-09 15:23:50.468086763 -0500
+++ crda-3.13/Makefile	2016-02-09 15:24:18.735406109 -0500
@@ -126,7 +126,6 @@ install-libreg:
 	$(NQ) '  INSTALL  libreg'
 	$(Q)mkdir -p $(DESTDIR)/$(LIBDIR)
 	$(Q)cp $(LIBREG) $(DESTDIR)/$(LIBDIR)/
-	$(Q)ldconfig
 
 %.o: %.c regdb.h $(LIBREG)
 	$(NQ) '  CC  ' $@
